src/main/java/spoon/reflect/visitor/ImportCleaner.java

```diff
@@ -85,7 +85,7 @@ protected void handleTargetedExpression(CtTargetedExpression<?, ?> targetedExpre
 			return;
 		}
 
-		if (target != null && target.isImplicit()) {
+		if (target.isImplicit()) {
```

Comment:
What about keeping `target != null` to avoid crashing with NPE on `target.isImplicit`?

Reply:
This check is redundant with the check just above([here](https://github.com/INRIA/spoon/blob/65e15161aacd2915b38e6b3ab8cd1d495a2c15be/src/main/java/spoon/reflect/visitor/ImportCleaner.java#L79)):
```java
if (target == null) {
   ...
   return;
}
```
If `target == null`, the condition `target.isImplicit()` will not be hit, isn't?